arch/xtensa/src/esp32s3/esp32s3_gpio.c

```diff
@@ -207,6 +208,17 @@ int esp32s3_configgpio(uint32_t pin, gpio_pinattr_t attr)
   func  = 0;
   cntrl = 0;
 
+  /* if pin 19 or 20 disable the USB/JTAG function and pull-up */
+
+  if (pin ==  19 || pin == 20)
+    {
+      uint32_t regval;
+      regval = getreg32(USB_SERIAL_JTAG_CONF0_REG);
+      regval &= ~(USB_SERIAL_JTAG_USB_PAD_ENABLE |
+       USB_SERIAL_JTAG_DP_PULLUP);
```

Comment:
Please align to let USB_SERIAL_JTAG_DP_PULLUP start exactly under USB_SERIAL_JTAG_USB_PAD_ENABLE to improve readability